pbst.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * @file
+ * @version 0.1
+ * @copyright 2017 CN-Consult GmbH
+ * @author Jens Stahl <jens.stahl@cn-consult.eu>
+ *
+ * License: Please check the LICENSE file for more information.
+ */
+
+require_once ("vendor/autoload.php");
+foreach (glob(getcwd()."/commands/*command.php") as $filename)
```

Comment:
This glob in the current working dir only is not optimal here.
It can be simply replaced by:
`foreach (glob(__DIR__."/commands/*command.php") as $filename) `